### PR TITLE
Blacklist incompatible codecs

### PR DESCRIPTION
**Pull request by dmlap**

### CODE REVIEW

File: src/videojs-hls.js [hunk 1]
```diff
@@ -585,7 +621,8 @@ videojs.HlsHandler.prototype.selectPlaylist = function () {
     effectiveBitrate,
     sortedPlaylists = this.playlists.master.playlists.slice(),
     bandwidthPlaylists = [],
-    i = sortedPlaylists.length,
+    now = +new Date(),
```
Comment (gkatsev) [inline src/videojs-hls.js]: `Date.now()` is supported in all browsers we care to support. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now#Browser_compatibility
Not a blocker.

Comment (dmlap) [inline src/videojs-hls.js]: Oops, forgot to address this one in my update. We should probably do a sweep to fix this all over the place.

Comment (gkatsev) [inline src/videojs-hls.js]: Not a big issue.

**Comment by imbcmdth** [comment]
One minor comment. Otherwise LGTM

**Pull request merged**